data/venue.yml

```diff
@@ -0,0 +1,21 @@
+name: Musuem of New Zealand, Te Papa, Tongarewa
+nickname: Te Papa
+address: 55 Cable Street, Te Aro, Wellington 6011, New Zealand
+url: https://www.tepapa.govt.nz/
+location:
+  lat: -41.2905022
+  long: 174.7817372
+maps:
+  url: https://goo.gl/maps/ZSehkwLEvk52
+  img_src_base: https://maps.googleapis.com/maps/api/staticmap?center=Te+Papa%2C+Wellington%2C+New+Zealand&maptype=roadmap
```

Comment:
I've set up a key and restricted it to `*.ruby.nz/*` - `AIzaSyBdI51q8kJ9s19RmWunLFFUZKFTxDXTSBA`